stk/backend/sputnik.py

```diff
@@ -298,245 +299,6 @@ def backward(ctx, dy):
 
 sdd = SDD.apply
 
-
-class SSD(torch.autograd.Function):
```

Comment:
Could you update the README to reflect that we've removed the doubly sparse kernels?

Reply:
I don't see changes to the README in the updated PR?